test/libsolidity/smtCheckerTests/typecast/upcast.sol

```diff
@@ -5,7 +5,7 @@ abstract contract D {}
 contract C {
```

Comment:
If we implement https://github.com/ethereum/solidity/issues/9939, most of the tests here will be redundant.